webpack.config.js

```diff
@@ -140,7 +161,7 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                         {
                             loader: 'file-loader',
                             options: {
-                                name: '/' + outputPath + '/fonts/[name].[hash].[ext]',
+                                name: outputPath + '/fonts/[name].[hash].[ext]',
```

Comment:
using absolute paths here was a workaround for the issue described in https://github.com/sulu/sulu/pull/6225/files#r708199344

when using an absolute path, the `file-loader` will return an absolute path and the `url()` inside of the `build/admin/main.HASH.css` **will not be resolved relative to the file**. the problem with this workaround is that it only works if the fonts are accessible via `domain.com/fonts/name.HASH.EXT`. 

for example, the workaround does not work when accessing a styleguide build inside of a subfolder like `jsdocs.sulu.io/2.3/`. in this case, the browser would try to load the font via `jsdocs.sulu.io/fonts/name.HASH.EXT` instead of `jsdocs.sulu.io/2.3/fonts/name.HASH.EXT`

Reply:
now that i think about it, the previous workaround probably causes issues when running sulu in a subfolder (https://github.com/sulu/sulu-docs/issues/495) too